### DISCUSSION TITLE
ReactOS and dotnet9x

### DISCUSSION THREAD

**Discussion by neil2ball** [itsmattkc/dotnet9x#34] [General]
I just ran your dotnet9x installer on a fresh copy of ReactOS 0.4.14 after watching your video. It seems to work fine there. I managed to locate two .NET Framework 2.0 apps and start them successfully. There were no bugs from my quick test.

I just finished watching your video. I admire your technique for backporting .NET Framework 2.0 to Windows 95. My money is on this sort of thing being the path forward for ReactOS and .NET compatibility.

**Comment by TheCustomFHD** [comment]
ReactOS targets XP/2003, so the official version of .NET or even Mono should work. Not sure how far they got with NT6+ support atm. Also, when testing stuff in ReactOS, please use the Nightly's, they are in way better shape.
